util/http/http_client.cc

```diff
@@ -97,5 +108,47 @@ bool Client::IsConnected() const {
   return socket_ && socket_->IsOpen();
 }
 
+///////////////////////////////////////////////////////////////////////////////
+
+SSL_CTX* SecureClient::CreateSslContext(SecurityLevel sec_level) {
+  SSL_CTX* ctx = SSL_CTX_new(TLS_client_method());
+  if (ctx) {
+    long tls_ver = sec_level == LEVEL_5 ? TLS1_3_VERSION : TLS1_2_VERSION;
```

Comment:
why not use tls1.3 hardcoded?

Reply:
We can, it use to be the default from the example I took to use TLS version 1.2, but I will change